Responsive-Web-Design-II MVP

willediger opened the pull request:
@gannondetroit

willediger committed:
added html css from responsive design-I, changed container width to 880px, adjusted margins to account for reduced width

willediger committed:
changed html to font-size 62.5%, changed font-sizes and letter-spacing to rems

willediger committed:
removed div in get started link to allow targeting width with percent. formatted splash widths with percents

willediger committed:
fixed font-size in body which cascaded down to paragraphs

willediger committed:
fixed layouts of 800px view to use % measurements on horizontal attributes

willediger committed:
finished styling 500px view with percents for horizontal measurements

willediger committed:
removed superfluous .container max-width statements that were used in development

willediger committed:
added stretch.html, and included transitions on all selectors to animate changes between breakpoints

GannonDetroit reviewed (commented):
Will, you have a bit of scroll bar action on your desktop version that appears when I get close to the breakpoint of your tablet view. The problem is most likely that you still have some pixel uses in your CSS (mainly for margins), I recommend using dev tools to figure out the percent number to convert those pixels to. Pixels create horizontal scroll bars, percents don't. (I only noticed this on the desktop view, your tablet and mobile views appeared solid.).

your transitions on the stretch is amazing. It looks like some professional Squarespace level stuff. Great job on that!!! 

(ignore previous review, that was meant for your Responsive Design I project).

willediger committed:
got rid of horizontal scroll bars on desktop view in lower widths by changing middle image to max width of 100%, and changing its margin-left to padding-left

willediger commented:
@GannonDetroit I fixed the horizontal scrollbar issue on the desktop view & lower widths. It wasn't caused by pixel margins, but by not having a max-width on the middle image. I set that to 100%, and changed the margin-left on the image to padding-left so that the width + padding of the image would have a max-width of 100%, rather than just the width of the image having a max-width 100% (which would still cause a horizontal scroll bar since the margin isn't taken into account by the max-width).
